mcr_scene_segmentation/common/include/mcr_scene_segmentation/scene_segmentation.h

```diff
@@ -7,36 +7,27 @@
 #ifndef MCR_SCENE_SEGMENTATION_SCENE_SEGMENTATION_H
 #define MCR_SCENE_SEGMENTATION_SCENE_SEGMENTATION_H
 
-#include <mcr_scene_segmentation/aliases.h>
 #include <mas_perception_libs/bounding_box.h>
-#include <vector>
-#include <pcl/filters/passthrough.h>
-#include <pcl/filters/voxel_grid.h>
-#include <pcl/filters/project_inliers.h>
+#include <mas_perception_libs/sac_plane_segmenter.h>
```

Comment:
Do we really need to split the segmentation definition? It becomes more complicated to understand the code, especially for new members, if we split many things. Since it's all about segmentation, it should be better and undertandable if we put all related methods here in this package. @sthoduka

Reply:
In my opinion and from experience, smaller classes are much easier to understand compared to a giant class definition.